subsystem/kubeapi_test.go

```diff
@@ -1370,6 +1370,41 @@ location = "%s"
 		}, "1m", "20s").Should(BeTrue())
 	})
 
+	It("Test update infraenv CR image type", func() {
+		By("Request infraenv")
+		infraEnvSpec.ClusterRef = nil
+		infraEnvSpec.CpuArchitecture = models.ClusterCPUArchitectureX8664
+		deployClusterDeploymentCRD(ctx, kubeClient, clusterDeploymentSpec)
+		deployAgentClusterInstallCRD(ctx, kubeClient, aciSpec, clusterDeploymentSpec.ClusterInstallRef.Name)
+		deployInfraEnvCRD(ctx, kubeClient, infraNsName.Name, infraEnvSpec)
+
+		By("Verify default image type requested on creation of infra-env")
+		infraEnvKey := types.NamespacedName{
+			Namespace: Options.Namespace,
+			Name:      infraNsName.Name,
+		}
+		var infraEnv *common.InfraEnv
+		Eventually(func() bool {
+			infraEnv = getInfraEnvFromDBByKubeKey(ctx, db, infraEnvKey, waitForReconcileTimeout)
+			infraEnvCrd := getInfraEnvCRD(ctx, kubeClient, infraEnvKey)
+			return *infraEnv.InfraEnv.Type == models.ImageTypeMinimalIso && models.ImageTypeMinimalIso == infraEnvCrd.Spec.ImageType
```

Comment:
I think this might need to be modified because of the recent changes
```suggestion
			return *infraEnv.InfraEnv.Type == models.ImageTypeMinimalIso
```

Reply:
which recent changes are you referring to?

Reply:
Related to this comment: https://github.com/openshift/assisted-service/pull/6998#discussion_r1860919042
You're no longer setting the infraenv spec.imageType in the code